src/server.ts

```diff
@@ -1,11 +1,17 @@
 'use strict';
 
 import express from 'express';
+import cors from 'cors';
+
+import { phonesRouter } from './routes/phones.router.js';
```

Comment:
You don't need to use separations in imports

```suggestion
import express from 'express';
import cors from 'cors';
import { phonesRouter } from './routes/phones.router.js';
```

Reply:
@ruslanpashkov I thought it's better if I separated logic blocks, but okay, I understand, thanks